src/Video.web.tsx

```diff
@@ -7,8 +7,24 @@ import React, {
   useState,
   type RefObject,
 } from 'react';
+import {unstable_createElement} from 'react-native-web';
```

Comment:
@zoriya Unfortunately `@types/react-native-web` doesn't export this. I guess I'll need to submit a PR to get that added there, or `@ts-ignore` this.

Reply:
I went for `@ts-expect-error` with a description for now.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72588

If the PR is accepted, the `@ts-expect-error` will start failing, and since this project doesn't have a lockfile that might be on some unrelated PR.

Maybe it makes sense to wait for the types PR?

Reply:
The new version of `@types/react-native-web` has been published, I've updated the package.json and removed the `@ts-expect-error`.